File: Week 1 - C/Problem Set 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int i = 1; i <= height; i++)
    {

        for (int j = height; j > i; j--)
        {
            printf(" ");
        }
        for (int s = 0; s < i; s++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week 6 - Python/Problem Set 6/sentimental-mario-less/mario.py
import cs50


def main():
    height = get_int()

    for i in range(height):
        for j in range(height - i - 1):
            print(" ", end="")

        for s in range(i + 1):
            print("#", end="")
        print("")


def get_int():
    while True:
        x = cs50.get_int("Height: ")
        if x > 0 and x < 9:
            break
    return x


main()
